### ISSUE TITLE
Ansible version sanity check needed?

### ISSUE THREAD

**Issue by ryanj** [2015-Middleware-Keynote/demo-ansible#139]
While we're waiting on #79, it might be good to add an ansible version sanity check to help users identify version mismatch issues.

Here is an example of a mis-reported error (my bad) that might have been avoided: #132

**Comment by thoraxe** [comment]
Good call. Should be pretty trivial to modify run.py to do "ansible --version" and examine the output ?
